lib/nacre/abstract_resource_collection.rb

```diff
@@ -1,3 +1,4 @@
+require 'nacre/abstract_collection'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.